docs/google-kubernetes-tutorial.md

```diff
@@ -16,127 +17,123 @@ This tutorial takes you through these steps:
+	kubectl apply -f ./manifests/tiller-rbac.yaml &&
+	helm init --service-account tiller --upgrade
 
-``` sh
-kubectl create serviceaccount tiller --namespace kube-system
-kubectl apply -f manifests/tiller-rbac.yaml
-helm init --service-account tiller --upgrade
-```
+It takes a minute for helm to initialize its server component (tiller):
 
+	watch "kubectl get pods --namespace kube-system | grep tiller"
 
-## Deploy TiDB to your Kubernetes cluster
+When you see `Running`, it's time to proceed to the next step!
 
-We can get the TiDB helm charts from the tidb-operator source repository.
+## Deploy TiDB Operator
 
-```sh
-git clone https://github.com/pingcap/tidb-operator
-cd tidb-operator
-git checkout gregwebs/kube-tutorial
-```
+The first TiDB component we are going to install is the TiDB Operator, using a Helm Chart.  The TiDB Operator is the management system that works with Kubernetes to bootstrap your TiDB cluster and keep it running. This step assumes you are in the `tidb-operator` working directory:
 
+	kubectl apply -f ./manifests/crd.yaml &&
+	kubectl apply -f ./manifests/gke-storage.yml &&
+	helm install ./charts/tidb-operator -n tidb-admin --namespace=tidb-admin
 
-Now we can run the TiDB operator and the TiDB cluster
+We can watch the operator come up with:
 
-```sh
-kubectl apply -f ./manifests/crd.yaml
-kubectl apply -f manifests/gke-storage.yml
-helm install charts/tidb-operator -n tidb-admin --namespace=tidb-admin
-```
+	watch kubectl get pods --namespace tidb-admin -o wide
 
-We can watch the operator come up with
+If you see `Running`, the next step is to launch a TiDB cluster!
 
-	watch kubectl get pods --namespace tidb-admin -o wide
+## Deploy your first TiDB Cluster
 
-Now with a single command we can bring-up a full TiDB cluster.
+Now with a single command we can bring-up a full TiDB cluster:
 
-	helm install charts/tidb-cluster -n tidb --namespace=tidb
+	helm install ./charts/tidb-cluster -n tidb --namespace=tidb
 
-Now we can watch our cluster come up
+It will take a few minutes to launch.  You can monitor the progress with:
 
 	watch kubectl get pods --namespace tidb -o wide
 
+When you see all pods `Running`, it's time to proceed forward!
 
 ## Connecting to TiDB
 
-Now lets connect to our MySQL database. This will connect from within the Kubernetes cluster.
+There can be a small delay between the pod being up and running, and the service being available.  When you see `demo-tidb` appear, the service is ready to connect to:
+
+	watch "kubectl get svc -n tidb"
+
+You can connect to the clustered service within the Kubernetes cluster:
 
 	kubectl run -n tidb mysql-client --rm -i --tty --image mysql -- mysql -P 4000 -u root -h $(kubectl get svc demo-tidb -n tidb --output json | jq -r '.spec.clusterIP')
 
-Now you are up and running with a distributed MySQL database!
+For debugging purposes, you can also establish a tunnel between an individual TiDB pod and your Google Cloud Shell.  For example:
```

Comment:
I think this is better and we should delete the above. Also should mention that mysql will not persist across sessions. It could be better to use a docker container command due to the lack of persistence since the container will always install if it isn't there yet.

Reply:
I will address.  Thank you!